.npmignore

```diff
@@ -9,3 +9,4 @@ docs
 src
 test
 gulpfile.js
+node_modules
```

Comment:
@anandsainath what's going on with this? `node_modules` shouldn't be have an impact here